src/Faker/Provider/nl_NL/Person.php

```diff
@@ -314,4 +314,33 @@ public static function prefix()
     {
         return static::randomElement(static::$prefix);
     }
+    
+    /**
+     * 11 proof idNumber
```

Comment:
please add a `@link` annotation to a Wikipedia-like description of this id number, and of the algo you used here.